lms/product/plugin/misc.py

```diff
@@ -31,11 +31,6 @@ def post_launch_assignment_hook(
 
     def accept_grading_comments(self, application_instance):
```

Comment:
As of right now this method is overridden by the blackboard implementation and it always returns `False`.

I reckon by the time we merge this that won't be the case.

Reply:
What is the current status of this?

Reply:
> I reckon by the time we merge this that won't be the case.

Yeah, not overiden by BB now.